langchain/retrievers/self_query/base.py

```diff
@@ -121,11 +117,7 @@ def _get_relevant_documents(
         return docs
 
     async def _aget_relevant_documents(
-        self,
-        query: str,
-        *,
-        run_manager: Optional[AsyncCallbackManagerForRetrieverRun],
-        **kwargs: Any,
+        self, query: str, *, run_manager: Optional[AsyncCallbackManagerForRetrieverRun]
```

Comment:
just noticing there's decent amount of classes where run_manager is typed as Optional, some where it has a default val of None. on base it's not optional and has no default.  guessing discrepancy is unintentional @hinthornw @nfcampos?

Reply:
will save updating that for a sep pr

Reply:
It should be required. I had originally incorrectly made optional